tests/tests_geomstats/test_aac.py

```diff
@@ -0,0 +1,79 @@
+from tests.conftest import Parametrizer, TestCase, np_backend
+from tests.data.aac_data import (
+    AACFrechetMeanTestData,
+    AACGGPCATestData,
+    AACRegressorTestData,
+    MaxIterTestData,
+)
+
+IS_NOT_NP = not np_backend()
+
+
+class _MeanEstimatorTest(TestCase):
+    def test_fit(self, estimator, X, expected):
+        estimate_ = estimator.fit(X).estimate_
+
+        dist = estimator.metric.dist(estimate_, expected)
+        self.assertAllClose(dist, 0.0)
+
+
+class _GeodesicEstimatorTest(TestCase):
+    def test_fit(self, estimator, X, atol):
+        estimator.fit(X)
+
+        mean = estimator.mean_
+        direc = estimator.components_[0]
+
+        new_geo = estimator.metric.total_space_metric.geodesic(
+            initial_point=mean, initial_tangent_vec=direc
+        )
+
+        dists = estimator.metric.point_to_geodesic_aligner.dist(
+            estimator.metric, new_geo, X
+        )
+        self.assertAllClose(dists, 0.0, atol=atol)
+
+
+class _RegressionTest(TestCase):
```

Comment:
This could go in GeodesicRegressionTest?

Reply:
That's the (medium term) goal. We still need to think a bit better on the learning methods tests to be able to abstract better.